src/main/java/frc/utils/digitalinput/chooser/ChooserDigitalInputConstants.java

```diff
@@ -0,0 +1,9 @@
+package frc.utils.digitalinput.chooser;
+
+public class ChooserDigitalInputConstants {
+
+	public static final boolean DEFAULT_STATE = false;
+	public static final String DEFAULT_STATE_STRING = "false";
```

Comment:
to make sure they are both the same, change the DEFAULT_STATE_STRING's value to DEFAULT_STATE as a String